projects/go-containerregistry/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://github.com/google/go-containerregistry"
+language: go
+primary_contact: "jason@chainguard.dev"
```

Comment:
Surprised no one from google wants these mails as it's hosted in google org

Reply:
@imjasonh we actually have a distribution list for this, which you should be on.  It gates the AWS project we use to test ECR stuff.

Reply:
Pretty sure it is `go-containerregistry@chainguard.dev`

Reply:
@mattmoor Do you want that email address added to cc's?

Reply:
Yeah, I think that makes sense